src/main/webapp/app/layouts/navbar/navbar.component.html

```diff
@@ -127,6 +127,12 @@
                             <span jhiTranslate="global.menu.admin.audits">Audits</span>
                         </a>
                     </li>
+                    <li *jhiHasAnyAuthority="['ROLE_SYS_ADMIN']">
+                        <a class="dropdown-item" routerLink="revisions" routerLinkActive="active" (click)="collapseNavbar()">
```

Comment:
The auth-guard is configured to allow PROJECT_ADMIN and SYS_ADMIN and UI is allowed for SYSTEM_ADMIN only